### PR TITLE
Add contributing file

### PR DESCRIPTION
**Pull request by tarnawski**

### CODE REVIEW

**Review by PorkOne — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+- **[PSR-2 Coding Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)** - Check the code style before committing changes. 
+- **Add tests!** - Your patch won't be accepted if it doesn't have tests. 
+- **Document any change in behaviour** - Make sure the `README.md` and any other relevant documentation are kept up-to-date. 
+- **Create feature branches** - Don't ask us to pull from your master branch. 
+- **One pull request per feature** - If you want to do more than one thing, send multiple pull requests. 
+- **Send coherent history** - Make sure each individual commit in your pull request is meaningful. If you had to make multiple intermediate commits while developing, please [squash them](http://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages) before submitting. 
+
+
+##Happy coding!
```
Comment (mateuszmaslanka) [inline CONTRIBUTING.md]: Missing space after `##`

**Review by mateuszmaslanka — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (tarnawski) [reply CONTRIBUTING.md]: :+1: Improved.

Comment (mateuszmaslanka) [reply CONTRIBUTING.md]: @tarnawski did you push fix? I can't see it.

Comment (tarnawski) [reply CONTRIBUTING.md]: There are no specifications for styling md file but I think empty line on the end of file will be helpful to view changes in PR. I added the new line but github not see difference, if you click edit file and   you could see raw file with changes

Comment (mateuszmaslanka) [reply CONTRIBUTING.md]: I have no permission to edit file so I cannot see this :(
Anyway I think we are talking about two different things. You mentioned space at the end and I meant space between `##` and `Happy coding!` in the last line :)

Comment (tarnawski) [reply CONTRIBUTING.md]: Ahh I see - now should be correct. Thanks!

**Review by partikus — APPROVED** [review]

**Pull request merged**